tests/click_tests/test_autolock.py

```diff
@@ -195,21 +195,22 @@ def test_autolock_passphrase_keyboard(device_handler: "BackgroundDeviceHandler")
             # click at "j"
             debug.click(CENTER_BUTTON)
         elif debug.model == "R":
-            # press "a"
-            debug.press_middle()
+            # just go right
+            debug.press_right()
         time.sleep(1.5)
 
     # Confirm the passphrase
     if debug.model == "T":
         debug.click(buttons.OK, wait=True)
         assert device_handler.result() == "mnF4yRWJXmzRB6EuBzuVigqeqTqirQupxJ"
     elif debug.model == "R":
-        debug.press_left()  # go to BACK
-        debug.press_middle()  # PRESS back
-        debug.press_left()  # go to ENTER
-        debug.press_middle()  # press ENTER
+        # NOTE: because of passphrase randomization we just input the passphrase
+        # this way instead of "manually" clicking the buttons.
+        # (It would be a pain to go back to MENU from random position and
+        # navigating to CONFIRM - which is not in scope for this test.)
+        debug.input("j" * 8)
```

Comment:
this test is useless without actually clicking.

instead, skip the test for model R

Reply:
no, sorry, I misunderstood what's going on here. using `input` this way is fine, but then please move the `assert result` to top level (bc it's now the same for both models) and modify the comment accordingly